app/Repositories/PostRepository.php

```diff
@@ -83,10 +83,35 @@ public function create(array $data, $signupId)
             'url' => $fileUrl,
             'caption' => $data['caption'],
             'status' => isset($data['status']) ? $data['status'] : 'pending',
-            'source' => $data['source'],
-            'remote_addr' => $data['remote_addr'],
+            'source' => isset($data['source']) ? $data['source'] : null,
+            'remote_addr' => isset($data['remote_addr']) ? $data['remote_addr'] : null,
```

Comment:
Not relevant for this PR, but this is another thing we should set internally on v3 endpoints! (With `$request->ip()` reading from `X-Forwarded-For` headers provided by whatever service is making the Rogue call).

Reply:
good call!